nlp/README.md

```diff
@@ -0,0 +1,22 @@
+## DACCS - NLP docker image
+
+This image is able tu run all the python files required for the nlp notbooks.
+It has an installation of tree-tagger and java librarie needed for tree-tagger by download and extraction.
+
+To build and run the project :
+
+### 1 .Build docker image
+```bash
+cd path/to/nlp
+docker build -t nlp .
+```
+### 2. Run the docker image 
+```bash
+sudo docker run -it --rm nlp bash
```

Comment:
**never** run docker command as sudo
alsi, replace the `nlp` tag with `nlp-pavics-jupyter-image`